apps/earn-protocol-landing-page/app/institutions/self-managed-vaults/selfManagedVaults.module.css

```
@@ -0,0 +1,93 @@
+}
+
+.integrationsBlock p {
+  color: var(--color-text-secondary)
+}
+
+.benefitsHeaderWrapper {
+  text-align: center;
+  padding: var(--spacing-space-small);
+  margin-bottom: var(--spacing-space-medium);
+  max-width: 95%;
+  margin: 0 auto;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-bottom



(lint/suspicious/noShorthandPropertyOverrides)